### PR TITLE
New Package: rdma-core-22.1

### PR DESCRIPTION
**Pull request by ArmedAviator**
This is a new package that _should_ bring the base RDMA and RoCE base packages to Void Linux.  I am not certain it is correctly done as it has alot of libraries as well involved.

Additionally, the end of the output has this message:

> => WARNING: rdma-core-22.1_1: libibumad.so.3 not found in common/shlibs!
> => WARNING: rdma-core-22.1_1: libibverbs.so.1 not found in common/shlibs!
> => WARNING: rdma-core-22.1_1: librdmacm.so.1 not found in common/shlibs!
> => WARNING: rdma-core-22.1_1: libmlx4.so.1 not found in common/shlibs!
> => WARNING: rdma-core-22.1_1: libmlx5.so.1 not found in common/shlibs!

### CODE REVIEW

**Comment by maxice8** [comment]
Ignore those warnings

File: srcpkgs/rdma-core/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'rdma-core'
+pkgname=rdma-core
+version=22.1
+revision=1
+build_style=cmake
+hostmakedepends="cmake gcc libnl3-devel eudev-libudev-devel make pkg-config python3-devel valgrind-devel"
```
Comment (maxice8) [inline srcpkgs/rdma-core/template]: remove the following

- cmake (it is automatically added by having build_style=cmake)
- gcc (part of base-devel so always included)
- make ((part of base-devel so always included)
- pkg-config (replace it with pkgconf)

Move the following to makedepends= 

- libnl3-devel
- eudev-libudev-devel
- python3-devel
- valgrind-devel

File: srcpkgs/rdma-core/template [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+short_desc="RDMA core userspace libraries and daemons"
+maintainer="Rich G <rich@richgannon.net>"
+license="GPL-3.0-or-later"
```
Comment (maxice8) [inline srcpkgs/rdma-core/template]: Wrong license, please check upstream for correct license

Comment (ArmedAviator) [reply srcpkgs/rdma-core/template]: Corrected in new commit.

File: srcpkgs/rdma-core/template (hunk 1 above)
Comment (ArmedAviator) [reply srcpkgs/rdma-core/template]: Thank you, this is my first time using Git and xbps templates.

New commit edited.

File: srcpkgs/rdma-core/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'rdma-core'
+pkgname=rdma-core
+version=22.1
+revision=1
+build_style=cmake
+hostmakedepends="pkgconf"
+makedepends="libnl3-devel eudev-libudev-devel python3-devel valgrind-devel"
+short_desc="RDMA core userspace libraries and daemons"
+maintainer="Rich G <rich@richgannon.net>"
+license="GPL-2.0-or-later, Linux-OpenIB, BSD-2-Clause-FreeBSD"
```
Comment (maxice8) [inline srcpkgs/rdma-core/template]: Please pick between GPL-2.0-or-later or Linux-OpenIB (It is dual licensed and you're the maintainer)

See https://github.com/linux-rdma/rdma-core/blob/master/COPYING.md for the licensing situation

**Comment by maxice8** [comment]
Please keep it all in one commit

File: srcpkgs/rdma-core/template (hunk 3 above)
Comment (ArmedAviator) [reply srcpkgs/rdma-core/template]: D'oh, that makes sense.

File: srcpkgs/rdma-core/template [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'rdma-core'
+pkgname=rdma-core
+version=22.1
+revision=1
+build_style=cmake
+hostmakedepends="pkgconf"
```
Comment (maxice8) [inline srcpkgs/rdma-core/template]: python3 is missing from hostmakedepends=

See travis for teh error

**Pull request closed without merging**